configure.ac

```diff
@@ -365,6 +365,9 @@ AC_SUBST(PLATFORM_AOT_SUFFIX)
 
 ## PLATFORM_AOT_SUFFIX not so simple for windows :-)
 
+: ${CFLAGS="-g -O3"}
```

Comment:
This won't work since we're setting CFLAGS in run-jenkins.sh which will override this.